scripts/globals/abilities/elemental_siphon.lua

```diff
@@ -43,8 +43,7 @@ function onUseAbility(player,target,ability)
         spiritEle = 3
     end;
     
-    local pEquipMods = player:getMod(MOD_ENHANCES_ELEMENTAL_SIPHON);
```

Comment:
I'm not sure why you're removing this? This is not a bonus from skill, but a separate modifier on items such as this: http://www.ffxiah.com/item/27439/beck-pigaches